MapboxGeocoder/MBPlacemarkScope.swift

```diff
@@ -24,48 +24,49 @@ extension PlacemarkScope: CustomStringConvertible {
                 scope.insert(.Neighborhood)
             case "address":
                 scope.insert(.Address)
+                
```

Comment:
I think it's useful to distinguish this if-else from the above series of if statements.